bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -467,15 +462,7 @@ <h3>Zusammen ist man weniger allein</h3>
         <circle class="wednesday-circle light-green circle3" cx="719.996" cy="123" r="372" fill="#D1FFEE" />
         <circle cx="719.996" cy="123" r="283" fill="#2AC3A2" />
       </svg>
-      <button class="animated-button">
-          {{ resp_img(
-            url='img/firefox/challenge-the-default/logo-ctd.svg',
-            optional_attributes={
-              'loading': 'lazy',
-              'alt': 'Challenge the default'
-           })
-          }}
-      </button>
+      <button class="animated-button ctd-logo-sprite"></button>
```

Comment:
Issue: this `<button>` has no descriptive text, which is an a11y issue. It also seems to only do something on hover (at least on desktop), so does it actually need to be a button still? Could it just be a `<div>`?

Either way, it would be good to have some descriptive text in here somehow?